admin_manual/configuration_server/occ_command.rst

```diff
@@ -355,20 +355,34 @@ File Operations
 
 The ``files:scan`` command scans for new files and updates the file cache. You 
 may rescan all files, per-user, a space-delimited list of users, and limit the 
-search path::
+search path. If not using ``--quiet``, statistics at the end of the scan will be shown::
 
  $ sudo -u www-data php occ  files:scan --help
    Usage:
-   files:scan [-p|--path="..."] [-q|--quiet] [--all] [user_id1] ... [user_idN]
+   files:scan [-p|--path="..."] [-q|--quiet] [-v|--verbose] [--all] [user_id1] ... [user_idN]
 
  Arguments:
    user_id               will rescan all files of the given user(s)
 
  Options:
-   --path (-p)           limit rescan to this path, eg. 
-   --path="/alice/files/Music", the user_id is determined by the path and the 
-   user_id parameter and --all are ignored
+   --path                limit rescan to the path given (*) 
    --all                 will rescan all files of all known users
+   --verbose             files and directories beeing processed are shown additionally during scanning
+
+When using ``--path``, the path must consist of following components::
+
+  "user_id/files/path" or "user_id/files/mount_name"
+
+where the term ``files`` is mandatory.
+
+(*) Example::
+
+  --path="/alice/files/Music"
+
+The user_id ``alice`` is determined implicit by the path component.
+
+The precedence for identifying users is: --path, --all and then [user_id] and
+exclude each other.
```

Comment:
I don't understand what this is trying to say. The "Usage" section seems to imply that a scan can be done of any or all of the various command parameters in the one run:
--path user1/files/folder1 user2
would scan folder1 of user1 and also all of the files for user2.
But this paragraph I think might be trying to say the --path, --all and [user_id] parameters are exclusive - only use 1 at a time?
Of course --all is in a sense exclusive anyway, because if you use --all then putting other --path or [user_id] will not add to the things scanned anyway.